packages/ERTP/src/typeGuards.js

```diff
@@ -77,6 +77,20 @@ export const AmountShape = harden({
   value: AmountValueShape,
 });
 
+/**
+ * To be used as a guard for an amount pattern argument, i.e., an argument which
```

Comment:
"guard" terminology is wrong. It is a pattern, to be used in argument guard position of a method guard. TODO fix.

Reply:
Done